applications/MeshingApplication/python_scripts/gradual_variable_interpolation_process.py

```diff
@@ -0,0 +1,90 @@
+import KratosMultiphysics
+import KratosMultiphysics.MeshingApplication as KratosMA
+
+def Factory(settings, model):
+    if not isinstance(settings, KratosMultiphysics.Parameters):
+        raise Exception("expected input shall be a Parameters object, encapsulating a json string")
+    return GradualVariableInterpolationProcess(model, settings["Parameters"])
+
+class GradualVariableInterpolationProcess(KratosMultiphysics.Process):
```

Comment:
Why not C¿?

Reply:
Due to tight project deadlines, we prioritized a quick Python implementation. However, migrating this to C++ for better performance is on our to-do list.